runtime/onert/backend/cpu/ops/FullyConnectedLayer.h

```diff
@@ -65,6 +67,11 @@ class FullyConnectedLayer : public ::onert::exec::IFunction
 
   ir::Activation _activation;
   std::unique_ptr<nnfw::cker::FCTempArena> _temp_arena;
+
+#ifdef USE_RUY_GEMV
+  bool _freed_weights = false;        // weights is already freed?
```

Comment:
(Optional) I think `_weights_freed` or `_is_weights_freed` would be better.

Reply:
`_is_weights_freed` looks better, thanks.